frontend/src/scenes/hog-functions/configuration/HogFunctionInputs.tsx

```diff
@@ -492,6 +498,22 @@ export function HogFunctionInputWithSchema({
                                             Supports templating
                                         </LemonButton>
                                     )}
+
+                                    {schema.type === 'email' && (
```

Comment:
I like the scoping but I think we should make it that all templateable inputs support both hog or liquid. This will be easier to manage long term